setup.py

```diff
@@ -1,25 +1,25 @@
 ##===---------- setup.py - dpctl.ocldrv interface -----*- Python -*-----===##
 ##
-##               Data Parallel Control Library (dpCtl)
+# Data Parallel Control Library (dpCtl)
 ##
-## Copyright 2020 Intel Corporation
+# Copyright 2020 Intel Corporation
 ##
-## Licensed under the Apache License, Version 2.0 (the "License");
-## you may not use this file except in compliance with the License.
-## You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 ##
-##    http://www.apache.org/licenses/LICENSE-2.0
+# http://www.apache.org/licenses/LICENSE-2.0
 ##
-## Unless required by applicable law or agreed to in writing, software
-## distributed under the License is distributed on an "AS IS" BASIS,
-## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-## See the License for the specific language governing permissions and
-## limitations under the License.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 ##
 ##===----------------------------------------------------------------------===##
 ###
-### \file
-### This file builds the dpctl and dpctl.ocldrv extension modules.
+# \file
+# This file builds the dpctl and dpctl.ocldrv extension modules.
 ##===----------------------------------------------------------------------===##
```

Comment:
I think we should use one # in all lines or do not touch it now.

Reply:
I think it is effect of black formatter. The rule is to have one # as a comment starter

Reply:
Reverted this changes.